.github/workflows/gradle-release.yml

```
@@ -77,3 +77,82 @@ jobs:
       - name: Git Push
         run: |
           git push
+
+  release_apk:
+    needs: create_git_tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get the Git Tag name from the previous job
+        run: echo "tagName ${{ needs.create_git_tag.outputs.tagName }}"
```

actionlint (1.7.4)

property "tagname" is not defined in object type {}

(expression)